src/main/resources/leonardo-example.conf

```diff
@@ -0,0 +1,141 @@
+# Example config file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and security certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
+  #applicationName = "leonardo"
+  dataprocDefaultRegion = "YOUR_REGION"  # the google region for your dataproc
+  leoGoogleProject = "GOOGLE_PROJECT"    # the name of the google project to use during cluster startup
+                                         # *this is not the project the cluster will be created in
+  clusterUrlBase = "https://YOUR_DOMAIN/notebookes" # the base url to access your cluster
+  #jupyterServerName = "jupyter-server"
+}
+
+# These are the cluster defaults that have already been configured.
+# You should only set them in your config if you want a different value
+#clusterDefaults {
+  #numberOfWorkers = 0            # Standard, non-preemptible workers; Can't be 1
+  #masterMachineType = "n1-standard-4"
+  #masterDiskSize = 500           # Minimum is 100
+  #workerMachineType = "n1-standard-4"
+  #workerDiskSize = 500           # Minimum is 100
+  #numberOfWorkerLocalSSDs = 0    # Can be 0-8
+  #numberOfPreemptibleWorkers = 0 # Can't be 1; Can only be > 1 when Number of workers is > 1
+#}
+
+# Database connection information
+mysql {
+  db {
+    url = "jdbc:mysql://YOUR_DB_HOST/leonardo?requireSSL=true&useSSL=true&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
+    user = "USER_NAME"
+    password = "PASSWORD"
+  }
+}
+
+# To use the swagger page, supply a service account id and realm
+#swagger {
+  #googleClientId = ""
+  #realm = ""
+#}
+
+# Secure, 2-way communication between leo and jupyter
+# Each section below will show the commands to generate the necessary files
```

Comment:
These are commands not configs, it might be a little confusing to have them in `leonardo-example.conf`.

Maybe it could instead live in a separate file `src/main/resources/generate_jupyter_secrets.sh.example`?

Reply:
i didn't want it to actually be a shell script. i left out the -subj so i didn't have to put in a bunch of info that they would need to change. Maybe I should just make a MD in the repo sort of like your gist doc? I can add a link to it in the example.conf before the section where they need to use that info.

Reply:
I like that!